### ISSUE TITLE
as of ver 1.0.9 weapon override issue

### ISSUE THREAD

**Issue by Omatkoicorko** [BarleyFlour/WrathVisualAdjustmentsMod#38]
last version of mod on latest mod manager - override weapon visualization still is not working for main character and companions

**Comment by weizhen199** [comment]
yes, aexs is ok, but others such as two hand slash does no effect.
